include/migration/migration.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MIGRATION_HPP_
+#define MIGRATION_HPP_
+
+#include <Poco/Data/Session.h>
+
+class Migration {
+public:
+    /**
+     * Creates database migrator instance.
+     *
+     * @param session database session.
+     */
+    explicit Migration(Poco::Data::Session& session, const std::string& migrationDir);
```

Comment:
I don't know, probably we use explicit for one parameter constructors only, right?

Reply:
Removed